bindings/node/addon/mongocrypt.cc

```diff
@@ -514,6 +514,10 @@ MongoCrypt::MongoCrypt(const CallbackInfo& info)
         mongocrypt_setopt_bypass_query_analysis(_mongo_crypt.get());
     }
 
+    if (options.Get("rangeV2").ToBoolean()) {
```

Comment:
What happens if rangeV2 is provided? Does it return a false? If it can return null an additional check would be nice.

Reply:
I don't think I _fully_ understand your question (probably a "not" missing somewhere?), but this is the equivalent of `if (!!options.rangeV2)` in JS – we do a similar check right above this one for `bypassQueryAnalysis`. This type of check is fine for boolean values, including optional ones.

Reply:
`Null` and `Undefined` are represented in Node-API as distinct values, not a C/C++ `nullptr`, so that's all good, and `.ToBoolean()` corresponds directly to the JS standard [`ToBoolean()`](https://tc39.es/ecma262/#sec-toboolean) operation 🙂